### PR TITLE
Complete unfinished sentence, apply style guide

### LINKED ISSUE #4549: Issue with k8s.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/
**This is a...** 

- [ ] Feature Request
- [x ] Bug Report

**Problem:**
Sentence not complete. The section **Building your own kubeconfig file**, last line is not completed. I think the text is missing

**Proposed Solution:**
The documentation has to be completed.

**Page to Update:**
https://kubernetes.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/

### PR DESCRIPTION
**Pull request by zacharysarah**
Closes #4549 

This PR:
- finishes the sentence fragment at the end of the heading
- applies style guide conventions to the heading

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4576)

### CODE REVIEW

File: docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md [hunk 1]
```diff
@@ -164,25 +164,24 @@ the settings relevant to the `current-context` by passing `--minify`. See
...
+{: .note}
 
-- Since the user for the current context is "green-user", any client of the api-server using this kubeconfig file would naturally be able to log in successfully, because we are providing the green-user's client credentials.
-- Similarly, we can operate as the "blue-user" if we choose to change the value of current-context.
+The sample kubeconfig file provides client credentials for the user `green-user`. Because the user for `current-context` is `green-user`, any client of the API server using the sample kubeconfig file could log in successfully. Similarly, we can operate as  `blue-user` by changing the value of `current-context`.
 
-In the above scenario, green-user would have to log in by providing certificates, whereas blue-user would just provide the token.  All this information would be handled for us by the
+In the example provided, `green-user` logs in by providing certificates, and blue-user  provides a token. Login information is specified with the `kubectl config set-credentials` command. For more information, see "[Commands for the example file](commands-for-the-example-file)".
```
Comment (chenopis) [inline docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md]: This link is broken.

File: docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md [hunk 2]
```diff
@@ -311,4 +310,4 @@ So, tying this all together, a quick start to create your own kubeconfig file:
 
 - Replace the snippet above with information for your cluster's api-server endpoint.
 
-- Make sure your api-server is launched in such a way that at least one user (i.e. green-user) credentials are provided to it.  You will of course have to look at api-server documentation in order to determine the current state-of-the-art in terms of providing authentication details.
+- Make sure your api-server is launched in such a way that at least one user (for example, green-user) credentials are provided to it.  You will of course have to look at api-server documentation in order to determine the current state-of-the-art in terms of providing authentication details.
```
Comment (chenopis) [inline docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md]: Do we want to be consistent with code highlighting `green-user`?

File: docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md (hunk 1 above)
Comment (zacharysarah) [inline docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md]: Good catch.

File: docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md (hunk 2 above)
Comment (zacharysarah) [inline docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig.md]: Yes--good catch.

**Comment by zacharysarah** [comment]
> While you're at it, could you put this doc into the Tasks template?

Will do. :+1:

**Comment by zacharysarah** [comment]
@chenopis Ready for review again! 🙇

**Comment by zacharysarah** [comment]
@chenopis 👋 

> I put in the Liquid template tags to show you an example of how to retrofit a doc into the Task template.

Thanks for this--sorry for missing what you meant! The example is helpful. 🙇

**Pull request merged**